README.md

```diff
@@ -1,4 +1,5 @@
 [![Marketplace Version](https://vsmarketplacebadge.apphb.com/version/redhat.vscode-quarkus.svg "Current Release")](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus)
+[![Build Status](https://travis-ci.org/redhat-developer/vscode-quarkus.svg?branch=master)](https://travis-ci.org/github/redhat-developer/vscode-quarkus)
```

Comment:
can you move it before the markeplace badge?

Reply:
fixed